ui/app/pages/swaps/swaps.util.js

```diff
@@ -323,6 +327,21 @@ export async function fetchSwapsFeatureLiveness() {
   return status?.active
 }
 
+export async function fetchSwapsQuoteRefreshTime(defaultValue = 0) {
+  const response = await fetchWithCache(
+    getBaseApi('refreshTime'),
+    { method: 'GET' },
+    { cacheRefreshTime: 600000 },
+  )
+
+  // We presently use milliseconds in the UI
+  if (response?.seconds) {
```

Comment:
While there's no stopping the API from giving a wrong answer, we should at least make a best-effort attempt to validate it. If we can avoid an API bug affecting our product that would be ideal.

e.g.
```suggestion
  if (typeof response?.seconds === 'number' && response.seconds > 0) {
```